src/base/rss/private/rss_parser.cpp

```diff
@@ -33,16 +33,329 @@
 
 #include <QDebug>
 #include <QDateTime>
+#include <QGlobalStatic>
+#include <QHash>
 #include <QMetaObject>
 #include <QRegExp>
 #include <QStringList>
 #include <QVariant>
+#include <QXmlStreamEntityResolver>
 #include <QXmlStreamReader>
 
 #include "../rss_article.h"
 
 namespace
 {
+    using StringHash = QHash<QString, QString>;
```

Comment:
seems you can inline this

Reply:
No. It contains comma, so it confuses preprocessor.